package.json

```diff
@@ -44,27 +44,35 @@
     "git-hooks": "^1.0.2",
     "grunt": "^0.4.5",
     "grunt-bump": "^0.7.0",
-    "grunt-casperjs": "^2.1.0",
     "grunt-contrib-concat": "^1.0.0",
     "grunt-contrib-connect": "^1.0.0",
     "grunt-contrib-uglify": "^1.0.0",
     "grunt-contrib-watch": "^1.0.0",
     "grunt-jscs": "^2.8.0",
     "grunt-mocha": "^0.4.15",
+    "grunt-webdriver": "just-boris/grunt-webdriver#infinite-loop",
```

Comment:
there is my fork. I hope the necessary fixes will be merged soon, and you can switch to original package: see https://github.com/webdriverio/grunt-webdriver/pull/80